### PR TITLE
Added Alchemy Support to the README Page 

### PR DESCRIPTION
**Pull request by smudgil**
### What does this PR do?
This PR adds Alchemy to the list of supported features on the README page.

### Where should the reviewer start?
Look at the README page, under the Features section

### Why is it needed?
Web3 users can use Alchemy as a remote node provider

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Features
     and Geth's
     [Personal](https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal)
     client APIs
+-   Support for [Alchemy](https://docs.alchemyapi.io/alchemy/guides/getting-started#web-3-j), so you don't have to run an Ethereum client yourself
```
Comment (conor10) [inline README.md:43]: Please can you  put Alchemy and Infura on the same line so we're not repeating ourselves?

**Review by conor10 — CHANGES_REQUESTED** [review]

**Review by smudgil — COMMENTED** [review]
Consolidated support for Alchemy and Infura in the same line

**Review by conor10 — CHANGES_REQUESTED** [review]
Please can you also add this to the Web3j docs too and I'll merge. https://github.com/web3j/web3j-docs

**Comment by smudgil** [comment]
> Please can you also add this to the Web3j docs too and I'll merge. https://github.com/web3j/web3j-docs

just updated it! :)

**Review by conor10 — APPROVED** [review]

**Pull request merged**